src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  --pf-c-scrollspy__list--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__list--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__list--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-scrollspy--m-vertical__list--before--BorderTopWidth: 0;
+
+  // with background
+  --pf-c-scrollspy--m-background--BackgroundColor: var(--pf-global--BackgroundColor--200);
+
+  // link
+  --pf-c-scrollspy__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--link--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-scrollspy__link--OutlineOffset: calc(-1 * var(--pf-global--spacer--sm));
+
+  // before
+  --pf-c-scrollspy__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy__link--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__link--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--before--BorderColor: transparent;
+  --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-scrollspy__item-text--Color: var(--pf-global--Color--200);
+  --pf-c-scrollspy__link--hover__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__link--focus__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__item--m-current__item-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-scrollspy__label--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__label--PaddingBottom: var(--pf-global--spacer--md);
+
+  padding-right: var(--pf-c-scrollspy--PaddingRight);
+  padding-left: var(--pf-c-scrollspy--PaddingLeft);
+  background-color: var(--pf-c-scrollspy--BackgroundColor);
+
+  &.pf-m-background {
+    --pf-c-scrollspy--BackgroundColor: var(--pf-c-scrollspy--m-background--BackgroundColor);
+  }
+
+  &.pf-m-center {
+    text-align: center;
```

Comment:
I would probably make the parent flex, and set `justify-content` to center the box, versus an inheritable property like `text-align` which may require resetting the alignment on nested children if we add more things that shouldn't be center aligned, since it applies to any child that is inline.